commit/operator/operator_noLUT.c

```diff
@@ -185,3 +185,62 @@ void COMMIT_At(
         pthread_join( threads[t], NULL );
     return;
 }
+
+void COMMIT_L(
```

Comment:
Why all these functions are empty if no LUT is selected? The regularization should be enforced in any case, no?

Reply:
It is in the TODO to enable the Tikhonov regularization when no LUT is selected.